src/super_gradients/training/utils/quantization/export.py

```diff
@@ -34,7 +34,17 @@ def export_quantized_module_to_onnx(model: torch.nn.Module, onnx_filename: str,
     # Export ONNX for multiple batch sizes
     logger.info("Creating ONNX file: " + onnx_filename)
     dummy_input = torch.randn(input_shape, device=next(model.parameters()).device)
-    torch.onnx.export(model, dummy_input, onnx_filename, verbose=False, opset_version=13, enable_onnx_checker=False, do_constant_folding=True)
+
+    if train:
+        training_mode = TrainingMode.TRAINING
+        model.train()
+    else:
+        training_mode = TrainingMode.EVAL
+        model.eval()
+
+    torch.onnx.export(
+        model, dummy_input, onnx_filename, verbose=False, opset_version=13, enable_onnx_checker=False, do_constant_folding=True, training=training_mode
```

Comment:
why do we need the training=training_mode? 
after we call prep_model_for_conversion the model can train anymore...

Reply:
For debug purposes we sometimes need an onnx file without optimizations
Moved prep model to eval mode only